DSCResources/MSFT_xPendingReboot/MSFT_xPendingReboot.psm1

```diff
@@ -57,33 +57,58 @@ Function Test-TargetResource
      param
     (
     [Parameter(Mandatory=$true)]
-    [string]$Name
+    [string]$Name,
+    [bool]$SkipComponentBasedServicing,
```

Comment:
Hi @TravisEz13 ,

Can you elaborate? By default, all the $skip* variables will be set to $false, so the if(-not $skip*) blocks will be entered, and the code should behave like it did before this PR.

Let me know if I missed something.